automation/startjanssendemo.sh

```diff
@@ -242,7 +239,7 @@ auth-server:
 EOF
 sudo helm repo add janssen https://docs.jans.io/charts
 sudo helm repo update
-sudo helm install janssen janssen/janssen -n jans -f override.yaml --kubeconfig="$KUBECONFIG" "$HELM_DEVELOPMENT_REPO"
+sudo helm install janssen janssen/janssen -n jans -f override.yaml --kubeconfig="$KUBECONFIG" --version="$JANS_VERSION"
```

Comment:
@misba7 have we handle empty `$JANS_VERSION`?

Reply:
empty JANS_VERSION installs latest, which was the default behavior.
I added the variable to be able to override it as needed.

Reply:
are you saying that `helm install --version=` work?

Reply:
ignore my last command, i have tested it. The `--version= <other-ops>` is working while `--version <other-ops>` is not.